app/utils/log/events.js

```diff
@@ -5,9 +5,9 @@ export default {
 	ONBOARD_CREATE_NEW_WORKSPACE_F: 'onboard_create_new_workspace_f',
 
 	// NEW SERVER VIEW
-	NEWSERVER_CONNECT_TO_WORKSPACE: 'newserver_connect_to_workspace',
-	NEWSERVER_CONNECT_TO_WORKSPACE_F: 'newserver_connect_to_workspace_f',
-	NEWSERVER_JOIN_OPEN_WORKSPACE: 'newserver_join_open_workspace',
+	NS_CONNECT_TO_WORKSPACE: 'ns_connect_to_workspace',
```

Comment:
These event keys are used on the rest of the app to log events to our Analytics like this.
`logEvent(events.NEWSERVER_CONNECT_TO_WORKSPACE);`

Since you're changing the key, you have to search for usage on the rest of the app and update there as well to something like:
`logEvent(events. NS_CONNECT_TO_WORKSPACE);`